ipa/ipa_cloud.py

```diff
@@ -140,7 +140,7 @@ def __init__(
         self.inject = self.ipa_config['inject']
         self.instance_type = self.ipa_config['instance_type']
         self.test_files = list(self.ipa_config['test_files'])
-        self.timeout = self.ipa_config['timeout']
+        self.timeout = int(self.ipa_config['timeout'])
```

Comment:
Is it possible for `self.ipa_config['timeout']` to be `None` ?

Reply:
No None is removed from command line args and there is a default value. https://github.com/SUSE-Enceladus/ipa/blob/master/ipa/ipa_cloud.py#L51